start-docker.sh

```diff
@@ -2,16 +2,16 @@
 docker stop $(docker ps -a -q)
 
 #Remove containers
-docker rm postgres-catalogues
-docker rm rso-catalogues
+#docker rm postgres-catalogues
+#docker rm rso-catalogues
```

Comment:
A ti ne pusti cel kup containerjev, če ne odstraniš starih? DB lahko laufa skoz, ampak rso-catalogues pa bo večkrat deployan, če testiraš, pa jih bo veliko?

Reply:
To pa niiti gledal nisem, bom preveril šeenkrat potem

Reply:
Okej vidm da tole ni kul ne...